build.gradle.kts

```diff
@@ -294,7 +294,6 @@ tasks.register("javaPreCommit") {
   dependsOn(":sdks:java:fn-execution:build")
   dependsOn(":sdks:java:harness:build")
   dependsOn(":sdks:java:harness:jmh:build")
-  dependsOn(":sdks:java:io:azure:build")
```

Comment:
this is already covered by `beam_PreCommit_Java_Azure_IO_Direct` that triggers for common trigger paths (e.g. java/core) so remove it from here.